web/src/components/apps/AppDetailPage.tsx

```diff
@@ -514,7 +514,6 @@ function AppDetailPage(props: Props) {
                   render={(renderProps) => (
                     <PreflightResultPage
                       logo={selectedApp.iconUri}
-                      app={selectedApp}
```

Comment:
this shows up as an error now that `PreflightResultPage` is typechecked. not sure if it was ever used